README.md

```diff
@@ -1,4 +1,4 @@
-ACL (Access Control List)
+acl
```

Comment:
In this context "acl" (lower-case, as defined in our style guide) is the name of the module itself, which is distinct from "ACL" (upper-case), the technology the module supports. ACL the technology doesn't get directly referenced until that Overview section you pointed out, so that's where the definition fits best. 

A bit confusing, I know, but hopefully the end user won't even need to notice the distinction.